nextcord/ext/commands/bot.py

```diff
@@ -901,6 +902,88 @@ def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
             # revert sys.modules back to normal and raise back to caller
             sys.modules.update(modules)
             raise
+    
+    def load_extensions(
+            self, 
+            names: List[str], 
+            *, 
+            package: Optional[str] = None,
+    ) -> List[str]:
+        """Loads all extensions provided in a list.
+        
+        Parameters
+        ----------
+        names: :type:`List[str]`
+            The names of all of the extensions to load.
+        package: :type:`Optional[str]`
+            The package name to resolve relative imports with.
+            This is required when loading an extension using a relative path, e.g ``.foo.test``.
+            Defaults to ``None``.
+
+        Returns
+        --------
+        A :type:`List[str]` that contains the names of all of the extensions
+        that loaded successfully.
+
+        Raises
+        --------
+        ExtensionNotFound
+            An extension could not be imported.
+        ExtensionAlreadyLoaded
+            An extension is already loaded.
+        NoEntryPointError
+            An extension does not have a setup function.
+        ExtensionFailed
+            An extension or its setup function had an execution error.
+        """
+        # TODO: Extra arguments
+
+        loaded_extensions: List[str] = []
+
+        for i_extension in range(len(names)):
+            try:
+                self.load_extension(names[i_extension], package=package)
+            except Exception:
+                raise
```

Comment:
Possibly add an argument to suppress errors? Could be a nice feature like ``load_extensions([..,..,..,], suppress_errors=True)`` where it loads all the ones it can and returns those while logging exceptions. Idk, need some feedback from others